### ISSUE TITLE
When my "full name" (of my user) contains accents, I can no longer add comments

### ISSUE THREAD

**Issue by Jerome-Herbinet** [nextcloud/richdocuments#2417]
**Describe the bug**
If my full name is "Jérôme Herbinet", I can no longer add a comment. However, if I remove the accents and my full name is "Jerome Herbinet", it works again.

**To Reproduce**
1. Put accents in the "full name".
2. Try to add comments

**Expected behavior**
I should be able to add comments whether or not there are accents in my "full name".

**Client details:**
 - OS: Ubuntu 20.04 LTS
 - Browser : Firefox 104.0
 - Device: Laptop

## Server details

**Operating system**: Ubuntu 20.04 LTS

**Web server:** Apache (fpm-fcgi)

**Database:** mysql 10.3.34

**PHP version:** 8.0.22

**Nextcloud version:** 24.0.4.1

**Nextcloud Office:** 
 - richdocuments: 6.2.0
 - richdocumentscode: 22.5.502

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
No errors appear in the logs.
```

#### Browser log
```
No errors appear in the browser console.
```

</details>

**Comment by Raudius** [comment]
Issue appears to be fixed in Collabora 22.6.1, so this issue should be fixed once the `richdocumentscode` app gets updated.

See: https://github.com/CollaboraOnline/richdocumentscode#richdocumentscode

**Comment by Jerome-Herbinet** [comment]
@Raudius do you know when the 22.6.1 will be released ? I've been searching for some milestones / roadmap, but I sadly didn't find any information about this version.
